### DISCUSSION TITLE
I can't see my personal account and my domains are gone!

### DISCUSSION THREAD

**Discussion by michaelshimeles** [vercel/community#6561] [Account Issue]
### Question

My personal account got converted to a hobby team account which is cool (and announced) but for some reasons, all the domains I bought under that personal account no longer show.

Is anyone else experiencing this and is their a fix? Need some of those domains!!!!

**Comment by amyegan** [comment]
Hey @michaelshimeles! You should be able to see all of your domains at [vercel.com/account/domains](https://vercel.com/account/domains)

You can learn more about how to find and move your domains in https://github.com/vercel/community/discussions/6298
